homeassistant/components/withings/const.py

```diff
@@ -55,6 +55,6 @@ class Measurement(StrEnum):
 
 SCORE_POINTS = "points"
 UOM_BEATS_PER_MINUTE = "bpm"
-UOM_BREATHS_PER_MINUTE = f"br/{const.TIME_MINUTES}"
+UOM_BREATHS_PER_MINUTE = f"br/{UnitOfTime.MINUTES}"
```

Comment:
If this is data from the integration i dont think it should use our constant.

Reply:
I agree 
```suggestion
UOM_BREATHS_PER_MINUTE = "br/min"
```

Reply:
sounds reasonable, will remove it at all (_even the old variant of using the `homeassistant.const`_)